dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task210702UpdateStructureTable.java

```diff
@@ -0,0 +1,121 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotmarketing.common.db.DotDatabaseMetaData;
+import com.dotmarketing.startup.AbstractJDBCStartupTask;
+import com.dotmarketing.util.Logger;
+import java.sql.SQLException;
+import java.util.List;
+
+/**
+ * Upgrade task used to add columns `icon` and `sort_order` in `structure` table if needed
+ */
+public class Task210702UpdateStructureTable extends AbstractJDBCStartupTask {
+
+    @Override
+    public boolean forceRun() {
+        try {
+            return !new DotDatabaseMetaData().hasColumn("structure", "icon") &&
+                    !new DotDatabaseMetaData().hasColumn("structure", "sort_order") ;
+        } catch (SQLException e) {
+            Logger.error(this, e.getMessage(),e);
+            return false;
+        }
+    }
+
+    /**
+     * Returns the upgrade SQL query for PostgreSQL.
+     *
+     * @return The SQL statement(s).
+     */
+    @Override
+    public String getPostgresScript() {
+        String query = "";
+        try {
+            if(!new DotDatabaseMetaData().hasColumn("structure", "icon")) {
```

Comment:
nitpick. but I think only one instance of  `DotDatabaseMetaData` will do  no need to  new DotDatabaseMetaData() everytime